icon/chainscore.go

```diff
@@ -568,7 +568,7 @@ var chainMethods = []*chainMethod{
 			{"ivoter", scoreapi.Integer, nil, nil},
 		},
 		nil,
-	}, icmodule.RevisionICON2R0, 0},
+	}, icmodule.RevisionICON2R0, icmodule.RevisionIISS4 - 1},
```

Comment:
Isn't it `RevisionPreIISS4`?

Reply:
In `RevisionPreIISS4`, `setRewardFundAllocation` is required because IISS3 is still working.